plugins/azure/test/conftest.py

```diff
@@ -52,6 +51,9 @@ def delete_resource_tag(self, tag_name: str, resource_id: str) -> bool:
     def update_resource_tag(self, tag_name: str, tag_value: str, resource_id: str) -> bool:
         return False
 
+    def list_with_retry(self, spec: AzureApiSpec, **kwargs: Any) -> List[Json]:
```

Comment:
I would consider this a private method of the real azure client - not a public API.